Jenkinsfile

```diff
@@ -40,6 +40,17 @@ def buildProduction(String stageName) {
   deleteDir()
 }
 
+def buildSim(String stageName) {
+  stage("Build Sim ${stageName}") {
+    if (env.BRANCH_NAME in ['develop', 'staging', 'master', 'beta', 'testMaestro']) {
+      if (stageName == 'ios' && params.IOS_BUILD_SIM) {
+        sh 'npm run prepare.ios'
+        sh "node -r sucrase/register ./scripts/deploy.ts edge ios ${BRANCH_NAME} sim"
```

Comment:
Could be just `sh "node -r sucrase/register ./scripts/deploy.ts edge ios-sim ${BRANCH_NAME}"`

Reply:
done